src/tracks/playabletrack/wavetrack/ui/WaveTrackView.h

```diff
@@ -257,7 +257,8 @@ struct AUDACITY_DLL_API ClipParameters
       const SelectedRegion &selectedRegion, const ZoomInfo &zoomInfo);
 
    double tOffset;
-   double rate;
+   double sampleRate;
```

Comment:
The commit that changes this name fails to compile SpectrumView.cpp and SpectrumCache.cpp